### PR TITLE
Fix bad tests

### PR DESCRIPTION
**Pull request by timokoesters**

### CODE REVIEW

File: tests/10apidoc/03events-initial.pl [hunk 1]
```diff
@@ -226,10 +226,6 @@ sub matrix_sync
    )->on_done( sub {
       my ( $body ) = @_;
 
-      assert_json_keys( $body, qw( account_data rooms presence next_batch ) );
```
Comment (valkum) [inline tests/10apidoc/03events-initial.pl:229]: next_batch is used in line 234/230 doesn't this cause a perl key error if perl has something like that?

Comment (timokoesters) [reply tests/10apidoc/03events-initial.pl:229]: Oops, you're right. next_batch is even required. Can I do `assert_json_keys( $body, qw( next_batch ) );` ? (I'm not sure what qw does - is it for exact matching?)

Comment (timokoesters) [reply tests/10apidoc/03events-initial.pl:229]: Pushed ^, I think it's correct

**Comment by timokoesters** [comment]
Btw, am I using the correct target branch in this PR?

**Comment by valkum** [comment]
Yeah its the correct branch.

File: tests/10apidoc/03events-initial.pl (hunk 1 above)
Comment (valkum) [reply tests/10apidoc/03events-initial.pl:229]: according to google ;) qw just quotes words. So that should work

**Pull request merged**